### DISCUSSION TITLE
Can you share scheduled tasks with other people?

### DISCUSSION THREAD

**Discussion by JohnMroz** [dbeaver/dbeaver#16921] [Q&A] [answered]
Hello everyone,

I am new to dBeaver at work and have been utilizing it to schedule monthly queries. Is it possible to share these scheduled tasks with other people? This would help in case I am on PTO or if anything needs to be transferred.

Thank you!

**Comment by ShadelessFox** [comment, marked as answer]
Hello @JohnMroz,

Unfortunately, you can't. Schedules tasks are registered through an OS-specific scheduler service, meaning that we don't store any information about their schedule.
